hashicorptools.gemspec

```diff
@@ -44,96 +44,96 @@ Gem::Specification.new do |s|
     "spec/hashicorptools_spec.rb",
     "spec/spec_helper.rb"
   ]
-  s.homepage = "http://github.com/woodhull/hashicorptools".freeze
-  s.licenses = ["MIT".freeze]
-  s.rubygems_version = "2.6.8".freeze
-  s.summary = "Wrappers for terraform and packer".freeze
+  s.homepage = "http://github.com/woodhull/hashicorptools"
+  s.licenses = ["MIT"]
+  s.rubygems_version = "2.4.8"
```

Comment:
Not really, probably from my env running an old version, will attempt regenerating gemspec with upgraded gems